src/tribol/physics/AlignedMortar.cpp

```diff
@@ -209,6 +208,7 @@ void ComputeAlignedMortarGaps( CouplingScheme const * cs )
 
    MeshData& mortarMesh = meshManager.GetMeshInstance( mortarId );
    MeshData& nonmortarMesh  = meshManager.GetMeshInstance( nonmortarId );
+   IndexType const numNodesPerFace = mortarMesh.m_numCellNodes;
```

Comment:
`m_numNodesPerCell` might be clearer than `m_numCellNodes`.

Reply:
Fixed. Good call. This is legacy naming. Your suggestion makes more sense.